tools/update-webpage.sh

```diff
@@ -32,6 +32,7 @@ DEBUGGER_MD="07.DEBUGGER.md"
 CODING_STANDARDS_MD="08.CODING-STANDARDS.md"
 EXT_REFERENCE_ARG_MD="09.EXT-REFERENCE-ARG.md"
 EXT_REFERENCE_HANDLER_MD="10.EXT-REFERENCE-HANDLER.md"
+EXT_REFERENCE_MODULE_MD="11.EXT-REFERENCE-MODULE.mpd"
```

Comment:
Thank you for wiring the doc, but there is a typo in the extension of filename: `mpd` -> `md`.